src/components/TextFieldComponent/TextFieldComponent.jsx

```diff
@@ -0,0 +1,49 @@
+import TextField from '@mui/material/TextField';
+import InputAdornment from '@mui/material/InputAdornment';
+import SearchIcon from '@mui/icons-material/Search';
+
+import './TextFieldComponent.scss';
+
+const TextFieldComponent = ()=> {
+
+    return <TextField 
+    id="filled-basic" 
+    placeholder="Ex.: Veggie Burger" 
```

Comment:
Whats is this hardcoded placeholder? Since its reusable components its cannot be hardcoded